Formula/racer.rb

```diff
@@ -0,0 +1,33 @@
+class Racer < Formula
+  desc "Rust Code Completion utility"
+  homepage "https://github.com/phildawes/racer"
+
+  url "https://github.com/phildawes/racer/archive/v1.1.0.tar.gz"
+  sha256 "f969e66d5119f544347e9f9424e83d739eef0c75811fa1a5c77e58df621e066d"
+  head "https://github.com/phildawes/racer.git"
+
+  depends_on "rust" => :build
+
+  resource "rust_source" do
+    url Formula["rust"].stable.url
```

Comment:
@ilovezfs The thing is that currently AFAIK only racer uses the rust sources and there's no general consensus that they should be available. BUT that said, it is WAAAAY better IMO to do things the way you suggested and keep the source of the rust you installed around instead of downloading a good guess but not knowing for sure that the versions match. It could even be behind a flag or something, (though rust takes ages to compile). I mean a lot of rust programmers use racer, because it's the only completion tool for the language (aside from paying intellij for their intellij-rust IDE/plugin/experiment), so it might be worth supporting by default for an extra ~52-55 MB, I mean mysql keeps almost half a gig of garbage around that I doubt very many people use.